infrastructure/docker/docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 version: '3'
 
 volumes:
-    mysql-data: {}
+    postgres-data: {}
+    # Uncomment to use MySQL
+    # mysql-data: {}
```

Comment:
Why not call it `database-data`?

Reply:
Because explicit is better than implicit 
CF: https://www.python.org/dev/peps/pep-0020/
---
More over, if you want to have PG + (MySQL | ES | Redis), it will be confusing